web/src/typeahead_helper.ts

```diff
@@ -306,6 +307,8 @@ export function sort_people_for_relevance(
     if (!current_stream) {
```

Comment:
The code just above here could be cleaned up in a follow-up; both to check `undefined` explicitly rather than falsey (weird with a theoretical ID of `0` here) and maybe also doing some `?` fanciness on the lines above. But not a thing to change for this PR.

Reply:
https://github.com/zulip/zulip/pull/30119